src/io/io.hpp

```diff
@@ -146,4 +148,16 @@ namespace wash {
      * @return const IOManager& 
      */
     IOManager& get_io();
+
+    const std::vector<std::vector<double>*> get_force_scalars();
```

Comment:
References instead of pointers?

e.g. `std::vector<std::vector<double>&>>`

Reply:
Sticking with pointers, for the moment at least